module/chat.js

```diff
@@ -1,72 +1,72 @@
 /**
```

Comment:
Seems to contain only whitespace changes which will break blame history. I would prefer not to do this especially when there is no other change